app/controllers/placements/providers/partner_schools/placements_controller.rb

```diff
@@ -0,0 +1,26 @@
+class Placements::Providers::PartnerSchools::PlacementsController < ApplicationController
+  before_action :set_provider, only: %i[index show]
+  before_action :set_partner_school, only: %i[index show]
```

Comment:
Could you add an empty line here to separate the `before_action` from the controller action?